static/js/pages/instances/InstanceList.tsx

```diff
@@ -215,19 +215,23 @@ const InstanceList: FC = () => {
         },
         {
           content: instance.state?.network?.eth0?.addresses
-            .filter((item) => item.family === "inet")
+            .filter(
+              (item) =>
+                item.family === "inet" && !item.address.startsWith("127")
+            )
             .map((item) => item.address)
-            .filter((address) => !address.startsWith("127"))
```

Comment:
This is the same as before. The old version seems easier to read to me. Same for the `inet6` below. Why change both?

Reply:
We are filtering the same array twice. Not very good from a performance point of view. What do we care about more, performance or readability? (Personally, I find it perfectly readable this way too...)

Reply:
ok, seems right.